src/main/java/ruslan/shastkiv/bookstore/controller/BookController.java

```diff
@@ -47,4 +48,9 @@ public BookDto updateBook(@PathVariable Long id, @RequestBody CreateBookRequestD
     public void deleteBook(@PathVariable Long id) {
         bookService.deleteBook(id);
     }
+
+    @GetMapping("/search")
+    public List<BookDto> search(BookSearchParametersDto searchParametersDto) {
```

Comment:
you can add pageable here

Reply:
> you can add pageable here

If you mean pagination, I'm on my way to learning about it. Once I cover that topic, I'll definitely add it here.